### PR TITLE
Add a stale bot configuration

### PR DESCRIPTION
**Pull request by beam2d**
This PR adds a configuration of the stale bot: https://github.com/probot/stale. The numbers of `daysUntilStale` and `daysUntilClose` are borrowed from the configuration used at https://github.com/fchollet/keras. We can change the numbers if they are not suited to our case; feel free to propose other numbers.

### CODE REVIEW

**Comment by okuta** [comment]
Should we create stale label?

**Comment by beam2d** [comment]
Thanks. Created `stale` label.
